earth_enterprise/src/SConstruct

```diff
@@ -480,6 +480,10 @@ gee_version_number = env.get_open_gee_version().get_short()
 
 
 # Check if the version of the default compiler is at least 4.8:
+if not env['CXX']:
+    print "Please install the default C++ compiler for your distribution."
```

Comment:
Grammar: Please, add a comma after 'Please'.

Reply:
I disagree on the grammar. The sentence in question is correct without an additional comma.

Reply:
Final verdict?

Reply:
Keep as is